src/explorer/DeploymentSlotTreeItem.ts

```diff
@@ -18,7 +18,7 @@ export class DeploymentSlotTreeItem extends SiteTreeItem {
     }
 
     public get label(): string {
-        return this.siteWrapper.slotName;
+        return `${this.siteWrapper.slotName} (${this.site.state})`;

```

Comment:
nit: Having all the web apps say (Running) makes the explorer look very cluttered.  What do you think about only displaying (Stopped) and omitting (Running) for apps that are on?